integration_tests/e2e/goal/updateGoal.cy.ts

```diff
@@ -48,6 +48,7 @@ context('Update a goal', () => {
     // When
     updateGoalPage //
       .clearGoalTitle()
+      .setTargetCompletionDateFromValuePreviouslySaveInGoal()
```

Comment:
Quite a long method name, but it does what it says on the tin?
Basically its a method that sets the target completion date radio button on the Update Goal page from the value that was previously set for the Goal